src/components/portrait/index.d.ts

```diff
@@ -0,0 +1,6 @@
+export { default as Portrait } from "./portrait";
```

Comment:
So as per our TypeScript styleguide we want the main component exported as default (https://github.com/Sage/carbon/blob/master/docs/typescript-styleguide.md), so it would be this: 

```typescript
export { default } from "./portrait";
export * from "./portrait";
export { default as PortraitGravatar } from "./portrait-gravatar";
export * from "./portrait-gravatar";
export { default as PortraitInitials } from "./portrait-initials";
export * from "./portrait-initials";
```